podcasts/End of Year/Stories/CompletionRateStory.swift

```diff
@@ -0,0 +1,129 @@
+        "\(Int(startedAndCompleted.percentage * 100))%"
+    }
+
+    var body: some View {
+        GeometryReader { geometry in
+            PodcastCoverContainer(geometry: geometry) {
+                StoryLabelContainer(geometry: geometry) {
+                    SubscriptionBadge(tier: subscriptionTier, displayMode: .gradient, foregroundColor: .black)
+                    StoryLabel(L10n.eoyYearCompletionRateTitle(percentageToDisplay), for: .title, geometry: geometry)
+                    StoryLabel(L10n.eoyYearCompletionRateSubtitle(startedAndCompleted.started, startedAndCompleted.completed), for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
+                }
+                .padding(.bottom, geometry.size.height * 0.06)
+
+                Spacer()
+
+                VStack {
+                    ZStack {
+                        Circle()
+                           .rotation(.degrees(-90))
+                           .stroke(Color(hex: "292B2E"), style: StrokeStyle(lineWidth: geometry.size.height * 0.02))
+                           .frame(width: geometry.size.height * 0.35, height: geometry.size.height * 0.35)
+
+                        LinearGradient(
+                        stops: [
+                        Gradient.Stop(color: Color(red: 0.25, green: 0.11, blue: 0.92), location: 0.00),
+                        Gradient.Stop(color: Color(red: 0.68, green: 0.89, blue: 0.86), location: 0.24),
+                        Gradient.Stop(color: Color(red: 0.87, green: 0.91, blue: 0.53), location: 0.50),
+                        Gradient.Stop(color: Color(red: 0.91, green: 0.35, blue: 0.26), location: 0.74),
+                        Gradient.Stop(color: Color(red: 0.1, green: 0.1, blue: 0.1), location: 1.00),
+                        ],
+                        startPoint: UnitPoint(x: -0.65, y: 0.5),
+                        endPoint: UnitPoint(x: 1.49, y: 0.5)
+                        )
+                        .frame(width: geometry.size.height * 0.4, height: geometry.size.height * 0.4)
+                        .rotationEffect(.degrees(-90))
+                        .mask(
+                            Circle()
+                               .rotation(.degrees(-90))
+                               .trim(from: 1 - startedAndCompleted.percentage, to: 1.0)
+                               .stroke(.red, style: StrokeStyle(lineWidth: geometry.size.height * 0.03))
+                               .scaleEffect(.init(width: -1, height: 1))
+                               .frame(width: geometry.size.height * 0.35, height: geometry.size.height * 0.35)
+                        )
+
+                        VStack(spacing: 0) {
+                            Text(percentageToDisplay)
+                            .font(.custom("DM Sans", size: geometry.size.height * 0.127).weight(.light))
+                            .foregroundColor(Color(red: 0.98, green: 0.98, blue: 0.99))
+
+                            Text("completion rate")
```

Comment:
L10n?

Reply:
Ops, done in 7d935120a099888648ad8bf55ffa24f34c910fe9